cinema_service/settings.py

```diff
@@ -40,12 +40,14 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "rest_framework",
+    "debug_toolbar",
```

Comment:
The 'debug_toolbar' is included in INSTALLED_APPS. Ensure it is removed or conditionally included for production environments.